app/ui/browser-modern/sagas/page-sagas.js

```diff
@@ -49,18 +50,25 @@ export default function*() {
   ];
 }
 
-function* createPageSession({ id, location, options }) {
+function* createPageSession({ id, location, options, withUI = true }) {
```

Comment:
Added bulk actions for creating / destroying standalone sessions without any UI. The usage of the UAS in these sagas is expected to change to avoid spamming posts, and highlight the session restore aspect more appropriately.